introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone.ipynb

```diff
@@ -2,7 +2,9 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #61.        "InputDataConfig": [
</pre><p>To reduce verbosity and lines counts, Can you export to a common object an re-use, shared fields from training_job_definition, sush as: InputDataConfig, OutputDataConfig, ResourceConfig, RoleArn</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3623/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I thought about that, but I then decided that it was acceptable to have a bit of redundancy, if it helps the customer. In my mind, it would be easier for the customer to have all training or tuning related config in a single section that they can quickly copy paste, instead of having to piece things together from different cells. </p>

Reply:
I understand your point, but also I am thinking, that re-using fields, makes more evident to the user, that a lot of the configuration of TuningJob is actually common with TrainingJob, so they can focus more on the "actual" tuning configs. But I trust your instinct on that.